### ISSUE TITLE
can i make it comercial and sell it

### ISSUE THREAD

**Issue by kranercc** [VenomXNL/XNLRankBar#6]
hi, i want to implement it into different frameworks and take profit of it, is that ok with you

**Comment by VenomXNL** [comment]
Hello, 

"Officially" the license has already been very clearly stated on Github here and on the "official FiveM release page" of my mod/script. I do recommend reading the main information page on github and the release page on FiveM itself first :)

-------
HOWEVER, (and this is really an important 'if'), IF you are making your **own framework** and this is just a (small) part of it, then this is okay. Obviously it should be just a fraction of your own framework with all it's features, user stats, jobs/ranking system etc.

IF however you are literally 'just' making small ports so it can for example be "implemented" into frameworks like ESX, VRp (and whatnot is out there these days). Then the answer is no.

The reason for this is that back in the day when I still did FiveM mods (and the "open-source/script" community wasn't completely messed up by people asking money for the most 'terrible/simple scripts') I did quite a lot of research in these scripts, spend lots of my own spare time in these scripts and decided to upload them in LUA for free (so others can actually use it as open-source) to learn from, instead of choosing a (closed source) C# release.

For me releasing these 'game mods' was/is a way to provide others of 'information' so they can build/learn from it. If others would 'just take it' and then 'port it for money', that would just be defeating the whole purpose of me uploading the scripts as open source. Plus that I'm personally strongly against the 'asking money for basic script mods concept', instead of actually helping each-other grow as "programming individuals".

So to summarize: IF you are indeed making your own (full) framework, and it's a (small) part of it, I would not mind it for this individual request.

IF it is to 'port it' and then basically 'take credit for the graphic/UI work': Nope.


NOTE:
You are of-course free to use mine as example (how it is/was originally intended) and write your own variant/version of it, and commercially profit from your own version :) I however just don't agree to the 'license part' where I've released it for free and others would make a profit from my work. (I DO realize others will do this anyways, but were possible I will of course file copyright/license violation claims).

**Comment by kranercc** [comment]
"and decided to upload them in LUA for free (so others can actually use it as open-source) to learn from, instead of choosing a (closed source) C# release." alright bro, issue closed
